Eat_and_Greet/src/comps/ChooseImageButton/ChooseImageButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { Image, TouchableOpacity, View, StyleSheet } from 'react-native';
+import React from 'react';
+
+interface ChooseImageProps {
+  onPress?: () => void;
+  buttonSize: number;
+  imagePath: string;
+  color?: string;
+}
+
+const ChooseImageButton = (props: ChooseImageProps) => {
+    const variablestyles = StyleSheet.create({
+        outerSizing: {
+            borderRadius: props.buttonSize,
+            height: props.buttonSize,
+            width: props.buttonSize,
+            justifyContent: 'center',
+        },
+        innerSizing: {
+            height: props.buttonSize - props.buttonSize/2.5,
+            width: props.buttonSize - props.buttonSize/2.5,
+            marginLeft: props.buttonSize/5,
+            marginBottom: props.buttonSize/10
+        },
+        coloring: {
+            backgroundColor: props.color, 
+        }
+    })
+    return (
+        <View style={[variablestyles.outerSizing, variablestyles.coloring, {position: 'relative'}]}>
+            <TouchableOpacity onPress={props.onPress}>
+                <Image style={[variablestyles.innerSizing, {position: 'relative'}]} source={require('../../../assets/images/uploadImage.png')}></Image>
```

Comment:
Since 'imagePath' is passed in as a prop with this path as the default, shouldn't we use the prop here?